src/Momento.Sdk/PreviewVectorIndexClient.cs

```diff
@@ -32,7 +32,7 @@ public PreviewVectorIndexClient(IVectorIndexConfiguration config, ICredentialPro
     {
         var loggerFactory = config.LoggerFactory;
         controlClient =
-            new VectorIndexControlClient(loggerFactory, authProvider.AuthToken, authProvider.ControlEndpoint);
+            new VectorIndexControlClient(config, authProvider.AuthToken, authProvider.ControlEndpoint);
```

Comment:
Piped the vector index config through to the `VectorIndexControlGrpcManager` but I'm not actually entirely sure if we want control clients to enable keepalives. 

They're not enabled in the JS cache control and vector index control clients, but I'd been enabling them in the Go and Python PRs along the way before I realized that.

Reply:
yep i think there's no reason control and data clients shouldn't be configured consistently for now